hudi-client/hudi-client-common/src/main/java/org/apache/hudi/index/bucket/ConsistentBucketIndexUtils.java

```diff
@@ -270,4 +273,24 @@ private static boolean recommitMetadataFile(HoodieTable table, FileStatus metaFi
     }
     return false;
   }
+
+  /**
+   * Initialize fileIdPfx for each data partition. Specifically, the following fields is constructed:
+   * - fileIdPfxList: the Nth element corresponds to the Nth data partition, indicating its fileIdPfx
+   * - partitionToFileIdPfxIdxMap (return value): (table partition) -> (fileIdPfx -> idx) mapping
+   *
+   * @param partitionToIdentifier Mapping from table partition to bucket identifier
+   */
+  public static Map<String, Map<String, Integer>> generatePartitionToFileIdPfxIdxMap(Map<String, ConsistentBucketIdentifier> partitionToIdentifier) {
```

Comment:
can also add uts for the method ?

Reply:
Sure, will add more tests~

Reply:
Added test in `TestConsistentBucketIdIdentifier.java`